src/pages/home/report/ReportActionsView.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
-import {View, Keyboard, AppState} from 'react-native';
+import {
+    Animated,
+    View,
+    Keyboard,
+    AppState,
+    Easing,
```

Comment:
Oops, unused import

Reply:
Missed the lint error, sorry. Amended on the last commit!
Thanks for the reviews.